mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -749,6 +778,7 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 		ARG 3 world
 		ARG 4 shapes
 	METHOD m_vcfvpihz checkDespawn ()V
+	METHOD m_vcplipih getLightLevelDependentMagicValue ()F
```

Comment:
magic... value? shouldn't it just be getLightLevelDependentValue? whats the point of magic here? is there something im missing?

Reply:
Yeah Magic is bad, I'll remove that. As far as i can tell it hold nothing

Reply:
swag